packages/itwin/tree-widget/src/e2e-tests/CategoriesTree.test.ts

```diff
@@ -6,212 +6,148 @@
-    });
-
-    test("search", async ({ page }) => {
-      await locateNode(treeWidget, "Equipment").waitFor();
-      await treeWidget.getByRole("button", { name: "Open" }).click();
-      await treeWidget.getByPlaceholder("Search...").fill("PipeSupport");
+  test("initial tree", async ({ page }) => {
+    // wait for element to be visible in the tree
+    await locateNode(treeWidget, "Equipment").waitFor();
+    await takeScreenshot(page, treeWidget);
+  });
 
-      // wait for non searched for nodes to disappear
-      await locateNode(treeWidget, "Equipment").waitFor({ state: "hidden" });
-      await takeScreenshot(page, treeWidget);
-    });
+  test("expanded tree node", async ({ page }) => {
+    const node = locateNode(treeWidget, "Equipment");
+    await node.getByRole("button", { name: "Toggle", includeHidden: true }).click();
 
-    test("header buttons overflow", async ({ page }) => {
-      await locateNode(treeWidget, "Equipment").waitFor();
-      await treeWidget.getByRole("button", { name: "Open" }).click();
-      await treeWidget.getByRole("button", { name: "More" }).click();
-      await page.locator(".tree-header-button-dropdown-container").waitFor();
-      await takeScreenshot(page, treeWidget);
-    });
+    // wait for node at the bottom to be visible/loaded
+    await locateNode(treeWidget, "Equipment - Insulation").waitFor();
+    await takeScreenshot(page, treeWidget);
+  });
 
-    test("shows outlines when focused using keyboard", async ({ page }) => {
-      // click to focus on node
-      const node = locateNode(treeWidget, "Equipment", 1);
-      await node.click();
-      const treeContainer = page.locator("#tw-tree-renderer-container");
+  test("node with active filtering", async ({ page }) => {
+    const node = locateNode(treeWidget, "Equipment", 0);
 
-      // focus on checkbox using keyboard
-      await page.keyboard.press("Tab");
+    // hover the node for the button to appear
+    await node.hover();
+    await node.getByRole("button", { name: "Apply filter" }).click();
 
-      // ensure checkbox is focused
-      const checkbox = node.locator(".tw-tree-node-checkbox");
-      await expect(checkbox).toBeFocused();
+    await locateInstanceFilter(page).waitFor();
+    await selectPropertyInDialog(page, "Code");
+    await selectOperatorInDialog(page, "Equal");
+    await selectValueInDialog(page, "Equipment - Insulation");
 
-      await takeScreenshot(page, node, { boundingComponent: treeContainer, expandBy: { top: 10, bottom: 10 } });
+    await page.getByRole("button", { name: "Apply" }).click();
 
-      // shrink panel
-      await expandStagePanel(page, "right", -100);
-      // scroll to the right side
-      await scrollTree(page, 10000, 0);
+    // wait for filtered children to appear
+    await locateNode(treeWidget, "Equipment - Insulation").waitFor();
+    await treeWidget.getByRole("button", { name: "Apply filter" }).waitFor();
```

Comment:
do we even need this? I think the widget has a numer of such buttons, so I'm not sure this serves any purpose